dbldatagen/schema_parser.py

```diff
@@ -132,7 +132,7 @@ def getTypeDefinitionParser(cls):
                 pp.delimitedList(pp.Group(ident + pp.Optional(colon) + pp.Group(type_expr)))) + r_angle
 
             # try to capture invalid type name for better error reporting
-            invalid_type = pp.Word(pp.alphas, pp.alphanums+"_", as_keyword=True)
+            invalid_type = pp.Word(pp.alphas, pp.alphanums+"_", asKeyword=True)
```

Comment:
they are probably getting rid of asKeyword "asKeyword and useRegex are retained for pre-PEP8 compatibility, but will be removed in a future release" https://pyparsing-docs.readthedocs.io/en/latest/pyparsing.html so the previously written version of this probably better

Reply:
i checked and it does not work in some versions of the Databricks runtime while the use of asKeyword does. 

They may change this in future but for maximum compatibility with existing versions - we need to use `asKeyword`